src/libs/OptionsListUtils.js

```diff
@@ -398,7 +398,11 @@ function getLastMessageTextForReport(report) {
                 reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE &&
                 ReportActionUtils.isMoneyRequestAction(reportAction),
         );
-        lastMessageTextFromReport = ReportUtils.getReportPreviewMessage(iouReport, lastIOUMoneyReport, true);
+        // Whether parent report is a 1:1 report
```

Comment:
This comment is not necessary. Also, this involves workspace chat as well, not only 1:1

Reply:
Fixed.